analyzers/src/SonarAnalyzer.Common/Rules/MethodsShouldNotHaveIdenticalImplementationsBase.cs

```diff
@@ -66,13 +77,28 @@ protected override void Initialize(SonarAnalysisContext context)
                                 messageArgs: GetMethodIdentifier(method).ValueText));
                         }
                     }
-                }, ClassDeclarationSyntaxKind);
-        }
+                }, SyntaxKinds);
 
         protected abstract IEnumerable<TMethodDeclarationSyntax> GetMethodDeclarations(SyntaxNode node);
```

Comment:
Also these abstract members should go  above the constructor and the properties, together with other abstract members.